pandas/core/window/expanding.py

```diff
@@ -170,92 +193,275 @@ def apply(
+    @doc(
+        doc_template,
+        window_method="expanding",
+        aggregation_description="minimum",
+        parameters=window_agg_numba_args_kwargs_parameters,
+        numpy_args_kwargs="",
+        agg_method="min",
+        other_see_also="",
+        notes=numba_notes,
+        examples="",
+    )
     def min(self, *args, engine=None, engine_kwargs=None, **kwargs):
         nv.validate_expanding_func("min", args, kwargs)
         return super().min(*args, engine=engine, engine_kwargs=engine_kwargs, **kwargs)
 
-    @Substitution(name="expanding")
-    @Appender(_shared_docs["mean"])
+    @doc(
+        doc_template,
+        window_method="expanding",
+        aggregation_description="mean",
+        parameters=window_agg_numba_args_kwargs_parameters,
+        numpy_args_kwargs="",
+        agg_method="mean",
+        other_see_also="",
+        notes=numba_notes,
+        examples="",
+    )
     def mean(self, *args, engine=None, engine_kwargs=None, **kwargs):
         nv.validate_expanding_func("mean", args, kwargs)
         return super().mean(*args, engine=engine, engine_kwargs=engine_kwargs, **kwargs)
 
-    @Substitution(name="expanding")
-    @Appender(_shared_docs["median"])
+    @doc(
+        doc_template,
+        window_method="expanding",
+        aggregation_description="median",
+        parameters=window_agg_numba_parameters,
+        numpy_args_kwargs="",
+        agg_method="median",
+        other_see_also="",
+        notes=numba_notes,
+        examples="",
+    )
     def median(self, engine=None, engine_kwargs=None, **kwargs):
         return super().median(engine=engine, engine_kwargs=engine_kwargs, **kwargs)
 
-    @Substitution(name="expanding", versionadded="")
-    @Appender(_shared_docs["std"])
+    @doc(
+        doc_template,
+        window_method="expanding",
+        aggregation_description="standard deviation",
+        parameters=dedent(
+            """
+        ddof : int, default 1
+            Delta Degrees of Freedom.  The divisor used in calculations
+            is ``N - ddof``, where ``N`` represents the number of elements.
+        """
+        ),
+        numpy_args_kwargs=numpy_args_kwargs,
+        agg_method="std",
+        other_see_also="numpy.std : Equivalent method for Numpy array.\n",
+        notes=dedent(
+            """
+        The default ``ddof`` of 1 used in :meth:`Series.std` is different
+        than the default ``ddof`` of 0 in :func:`numpy.std`.
+
+        A minimum of one period is required for the rolling calculation.
+        """
+        ),
+        examples=dedent(
+            """
+        >>> s = pd.Series([5, 5, 6, 7, 5, 5, 5])
+
+        >>> s.expanding(3).std()
+        0         NaN
+        1         NaN
+        2    0.577350
+        3    0.957427
+        4    0.894427
+        5    0.836660
+        6    0.786796
+        dtype: float64
+        """
+        ),
+    )
     def std(self, ddof: int = 1, *args, **kwargs):
```

Comment:
OT i don't think *args should be used after the keyword argument ddof, mypy normally complains about the function receiving multiple values for argument.